Timezone/UserBasedTimezoneDetector.php

```diff
@@ -51,7 +51,5 @@ public function getTimezone()
         if ($user instanceof User) {
             return $user->getTimezone();
         }
-
-        return;
```

Comment:
hmm i thought that ``return;`` will return ``null``, but if this is correct what you say, it is valid to remove this line